app/models/course.rb

```diff
@@ -812,4 +813,10 @@ def services
       Courses::ContentStatusService.new
     end
   end
+
+  def validate_all_sites_publishable
+    if sites.map(&:urn).include?(nil)
+      errors.add(:sites, "must have a Unique Reference Number (URN)")
```

Comment:
<img width="1058" alt="Screenshot 2021-06-30 at 15 08 19" src="https://user-images.githubusercontent.com/616080/123975080-04563a80-d9b5-11eb-96ca-cd50aa4d49de.png">
 I think this message needs to be set in the translation similar to https://github.com/DFE-Digital/teacher-training-api/pull/2022/files#diff-44438ce218f5287c58d0017f965d888715635d94280669896f75841fbd7b4cd7R123

Reply:
All done 👍